plugins/modules/monitoring/datadog/datadog_monitor.py

```diff
@@ -330,7 +336,8 @@ def install_monitor(module):
         "locked": module.boolean(module.params['locked']),
         "require_full_window": module.params['require_full_window'],
         "new_host_delay": module.params['new_host_delay'],
-        "evaluation_delay": module.params['evaluation_delay']
+        "evaluation_delay": module.params['evaluation_delay'],
+        "include_tags": module.boolean(module.params['include_tags'])
```

Comment:
It is already a boolean:
```suggestion
        "include_tags": module.params['include_tags'],
```